include/libcyphal/presentation/subscriber.hpp

```diff
@@ -90,13 +94,25 @@ class SubscriberBase : public SubscriberImpl::CallbackNode  // NOSONAR cpp:S4963
 
 }  // namespace detail
 
-// TODO: docs
+/// @brief Defines a custom strong-typed message subscriber class.
+///
+/// Although the subscriber class is not requiring specifically Nunavut tool generated message type,
```

Comment:
nit: `is not requiring specifically` --> `does not specifically require a Nunavut tool generated message type`